test/L2Notifier.test.ts

```diff
@@ -58,6 +60,7 @@ const setup = async (): Promise<Contract[]> => {
 
 describe('L2Notifier', () => {
   beforeEach(async () => {
+    await sleep(1000);
```

Comment:
Add sleep because arbitrum were spammed error in production for that reason: "Too Many Requests error received from arbitrum-mainnet.infura.io" Not sure if sleep is the best approach for this error, but I saw it can fix this problem.